apps/meteor/server/methods/deleteUser.ts

```diff
@@ -49,6 +49,19 @@ Meteor.methods<ServerMethods>({
 			});
 		}
 
+		if (user.federated) {
```

Comment:
We have a helper to test this, it would be good to use this... https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/core-typings/src/IUser.ts#L196